### PR TITLE
Release notes for InfluxDB 2.7.11

### PR DESCRIPTION
**Pull request by gunnaraasen**
- Release notes for InfluxDB OSS 2.7.11.

- [x] Rebased/mergeable

### CODE REVIEW

**Review by davidby-influx — APPROVED** [review]
LGTM

**Review by devanbenz — APPROVED** [review]
LGTM

File: content/influxdb/v2/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -8,6 +8,30 @@ menu:
...
 
+## v2.7.11 {date="2024-11-25"}
+
+### Features
+
+- Add startup logging on the percentage of shards opened.
+- Improve error messages and logging during shard opening.
+- Add additional logging when loading field set changes.
+- Add `--pid-file` option to write a PID file on startup.
+- Add `--storage-wal-flush-on-shutdown` to flush WAL on database shutdown.
```
Comment (jstirnaman) [inline content/influxdb/v2/reference/release-notes/influxdb.md:19]: @jstirnaman New v2.7.11 options

**Review by jstirnaman — APPROVED** [review]
LGTM. I'll create an issue to document the new options.

File: content/influxdb/v2/reference/release-notes/influxdb.md [hunk 2]
```diff
@@ -8,6 +8,30 @@ menu:
 weight: 101
 ---
 
+## v2.7.11 {date="2024-11-25"}
```
Comment (devanbenz) [inline content/influxdb/v2/reference/release-notes/influxdb.md:11]: Please change date to today's date

**Review by devanbenz — CHANGES_REQUESTED** [review]

**Review by devanbenz — APPROVED** [review]

**Pull request merged**